PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -238,59 +238,64 @@ Usage: PreeFreeSurferPipeline.sh [options]
   --fmapmag=<file path>             Siemens Gradient Echo Fieldmap magnitude file
   --fmapphase=<file path>           Siemens Gradient Echo Fieldmap phase file
   --fmapgeneralelectric=<file path> General Electric Gradient Echo Field Map file
-									Two volumes in one file
-									1. field map in deg
-									2. magnitude
+                                    Two volumes in one file
+                                    1. field map in deg
+                                    2. magnitude
   --echodiff=<delta TE>             Delta TE in ms for field map or "NONE" if
-									not used
+                                    not used
   --SEPhaseNeg={<file path>, NONE}  For spin echo field map, path to volume with
-									a negative phase encoding direction (LR in
-									HCP data), set to "NONE" if not using Spin
-									Echo Field Maps
+                                    a negative phase encoding direction (LR in
+                                    HCP data), set to "NONE" if not using Spin
+                                    Echo Field Maps
   --SEPhasePos={<file path>, NONE}  For spin echo field map, path to volume with
-									a positive phase encoding direction (RL in
-									HCP data), set to "NONE" if not using Spin
-									Echo Field Maps
+                                    a positive phase encoding direction (RL in
+                                    HCP data), set to "NONE" if not using Spin
+                                    Echo Field Maps
   --seechospacing=<seconds>         Effective Echo Spacing of Spin Echo Field Map,
-									(in seconds) or "NONE" if not used
+                                    (in seconds) or "NONE" if not used
   --seunwarpdir={x,y,NONE}          Phase encoding direction (according to the *voxel* axes)
              or={i,j,NONE}          of the spin echo field map. 
                                     (Only applies when using a spin echo field map.)
   --t1samplespacing=<seconds>       T1 image sample spacing, "NONE" if not used
   --t2samplespacing=<seconds>       T2 image sample spacing, "NONE" if not used
   --unwarpdir={x,y,z,x-,y-,z-}      Readout direction of the T1w and T2w images (according to the *voxel axes)
            or={i,j,k,i-,j-,k-}      (Used with either a gradient echo field map 
-									or a spin echo field map)
+                                    or a spin echo field map)
   --gdcoeffs=<file path>            File containing gradient distortion
-									coefficients, Set to "NONE" to turn off
+                                    coefficients, Set to "NONE" to turn off
   --avgrdcmethod=<avgrdcmethod>     Averaging and readout distortion correction
-									method. See below for supported values.
+                                    method. See below for supported values.
+  --custombrain=(NONE|MASK|CUSTOM)  Whether to use a custom brain mask (MASK) the 
+                                    pre-existing custom brain images (CUSTOM)
```

Comment:
The terminology here is confusing to me.  I don't think users are going to know what "pre-existing" means.  Where would they go to get more details?

Reply:
Also, the default value is not specified in the usage statement.

Reply:
These are easy fixes.

Reply:
The decsription will now be read:
```
  [--custombrain=(NONE|MASK|CUSTOM)]  If PreFreeSurfer has been run before and you have created a custom
                                      brain mask saved as "<path>/T1w/custom_acpc_dc_restore_mask.nii.gz", specify "MASK". 
                                      If PreFreeSurfer has been run before and you have created custom T1w images to
                                      be used when peforming Atlas registration, specify "CUSTOM".
                                      When "MASK" or "CUSTOM" is specified, all the steps until Atlas registration 
                                      are skipped.
                                      If the parameter is ommited or set to NONE (the default), 
                                      standard image processing will take place.
```

@mharms Would that be clear?

Reply:
see my review below

Reply:
You specify the file name that gets used for the MASK condition, but not the CUSTOM condition.

Reply:
The specific file names were added to the parameter description for CUSTOM option.